spec/abci++/abci++_methods_002_draft.md

```diff
@@ -61,22 +61,23 @@ title: Methods
-    | validators       | repeated [ValidatorUpdate](#validatorupdate)                                                                                         | Initial genesis validators, sorted by voting power. | 4            |
-    | app_state_bytes  | bytes                                                                                                                                | Serialized initial application state. JSON bytes.   | 5            |
-    | initial_height   | int64                                                                                                                                | Height of the initial block (typically `1`).        | 6            |
+    | time                | [google.protobuf.Timestamp](https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Timestamp)   | Genesis time                                                  | 1            |
+    | chain_id            | string                                                                                                                                 | ID of the blockchain.                                         | 2            |
+    | consensus_params    | [ConsensusParams](#consensusparams)                                                                                                    | Initial consensus-critical parameters.                        | 3            |
+    | validator_set       | [ValidatorSetUpdate](#validatorsetupdate)                                                                                              | Initial genesis validator set update, sorted by voting power. | 4            |
+    | app_state_bytes     | bytes                                                                                                                                  | Serialized initial application state. JSON bytes.             | 5            |
+    | initial_height      | int64                                                                                                                                  | Height of the initial block (typically `1`).                  | 6            |
+    | initial_core_height | int64                                                                                                                                  | Height of the initial core height.                            | 7            |
 
 * **Response**:
 
     | Name             | Type                                         | Description                                     | Field Number |
-    |------------------|----------------------------------------------|-------------------------------------------------|--------------|
+------------------|------------------|----------------------------------------------|-------------------------------------------------|--------------|
```

Comment:
I think this should actually be (it doesn't render the table properly currently):
```suggestion
|------------------|----------------------------------------------|-------------------------------------------------|--------------|
```